### PR TITLE
UI responsiveness

### PR DESCRIPTION
**Pull request by hassanabbas16**
complete ui responisveness and pages rework.

home page:
home-phone
home-desk

fleet page:
fleet-phone
fleet-desk

subscription page:
subscription-phone
subscription-desk

autocare page:
autocare-desk
autocare-phone

about page:
about-phone
about-desk

contact page:
contact-phone
contact-desk

### CODE REVIEW

File: .idea/Fast-Clean-Service.iml
Comment (fizo-neechan) [inline .idea/Fast-Clean-Service.iml:1]: What is this for, if no use then remove these

File: .idea/vcs.xml
Comment (fizo-neechan) [inline .idea/vcs.xml:1]: same as above

File: .idea/inspectionProfiles/Project_Default.xml
Comment (fizo-neechan) [inline .idea/inspectionProfiles/Project_Default.xml:1]: same as above

File: .idea/modules.xml
Comment (fizo-neechan) [inline .idea/modules.xml:1]: same as above

File: .idea/.gitignore
Comment (fizo-neechan) [inline .idea/.gitignore:1]: Remove if unnecessary

File: .idea/Fast-Clean-Service.iml
Comment (hassanabbas16) [reply .idea/Fast-Clean-Service.iml:1]: needed

**Pull request merged**